views/calendar/calendar.js

```diff
@@ -126,134 +126,37 @@ window.calendarView = class calendarView extends View {
-                        </span><br>
-                    `);
-                        $(`#Sep`).append(`
-                        <span>${AllMonths.Sep[i].day} ${getMonth(AllMonths.Sep[i].date)}
-                        ${AllMonths.Sep[i].year} <a href="https://www.wikitree.com/wiki/${AllMonths.Sep[i].id}">
-                        ${AllMonths.Sep[i].name}</a> ${AllMonths.Sep[i].text} ${
-                            AllMonths.Sep[i].marriage
-                        } [<span class="small"><a href="https://www.wikitree.com/treewidget/${
-                            AllMonths.Sep[i].id
-                        }/6">share tree image</a>]
-                        </span><br>
-                    `);
-                        $(`#Oct`).append(`
-                            <span>${AllMonths.Oct[i].day} ${getMonth(AllMonths.Oct[i].date)}
-                            ${AllMonths.Oct[i].year} <a href="https://www.wikitree.com/wiki/${AllMonths.Oct[i].id}">
-                            ${AllMonths.Oct[i].name}</a> ${AllMonths.Oct[i].text} ${
-                            AllMonths.Oct[i].marriage
-                        } [<span class="small"><a href="https://www.wikitree.com/treewidget/${
-                            AllMonths.Oct[i].id
-                        }/6">share tree image</a>]
-                        </span><br>
-                            `);
-                        $(`#Nov`).append(`
-                        <span>${AllMonths.Nov[i].day} ${getMonth(AllMonths.Nov[i].date)}
-                        ${AllMonths.Nov[i].year} <a href="https://www.wikitree.com/wiki/${AllMonths.Nov[i].id}">
-                        ${AllMonths.Nov[i].name}</a> ${AllMonths.Nov[i].text} ${
-                            AllMonths.Nov[i].marriage
-                        } [<span class="small"><a href="https://www.wikitree.com/treewidget/${
-                            AllMonths.Nov[i].id
-                        }/6">share tree image</a>]
-                        </span><br>
-                    `);
-                        $(`#Dec`).append(`
-                        <span>${AllMonths.Dec[i].day} ${getMonth(AllMonths.Dec[i].date)}
-                        ${AllMonths.Dec[i].year} <a href="https://www.wikitree.com/wiki/${AllMonths.Dec[i].id}">
-                        ${AllMonths.Dec[i].name}</a> ${AllMonths.Dec[i].text} ${
-                            AllMonths.Dec[i].marriage
-                        } [<span class="small"><a href="https://www.wikitree.com/treewidget/${
-                            AllMonths.Dec[i].id
-                        }/6">share tree image</a>]
-                        </span><br>
-                    `);
-                    } catch {}
+                for (const monthName of MONTHS) {
+                    // loop over all months
+                    const shortName = monthName.slice(0, 3);
+                    let events = AllMonths[shortName];
+                    $(`#${shortName}`).append(
```

Comment:
Does this actualy work? 😁 - I've just written something out of head, without testing it 😁.

Reply:
I think I had to make a small tweak to it - but it worked!